### DISCUSSION TITLE
Add search option with local data in html

### DISCUSSION THREAD

**Discussion by benzen** [fomantic/Fomantic-UI#2565] [Behaviors (Api, Form Validation, Visibility)] [answered]
Currently we have an option of local search with data that is inserted through js code.
What I think would be a nice alternative would be the same, but with data that is already present in the html.
just like we already have an option of pre-rendered data from dropdown and popup.

**Comment by lubber-de** [comment, marked as answer]
How would you expect the searchable data as DOM nodes being structured? dropdown and popup are HTML markup themselves, but data for search is always an object.
So, you can at least already use the `source` setting to provide an already existing data object

https://fomantic-ui.com/modules/search.html#/usage

```javascript
var
  content = [
    {
      title: 'Horse',
      description: 'An Animal',
    },
    {
      title: 'Cow',
      description: 'Another Animal',
    }
  ]
;
$('.ui.search')
  .search({
    source : content,
    searchFields   : [
      'title'
    ],
    fullTextSearch: false
  })
;
```
